### PR TITLE
Fixed justwatch

### PR DESCRIPTION
**Pull request by barsa2000**
I rewrote the justwatch client to use the new graphql queries, it still needs some refactoring but it works :)

also the tmdb search in sonarr now always returns `None, None` because I don't have time to fix it now

### CODE REVIEW

**Comment by barsa2000** [comment]
I should have also fixed the TMDB search and the sonarr episode watch status update functions (they were broken because we were using deprecated APIs)

**Comment by barsa2000** [comment]
Also just FYI I think we no longer need the gpl3 license with this PR, I removed the justwatch library you were using, we could go back to MIT

**Comment by barsa2000** [comment]
bump, please check this PR so I can work on other stuff @TheArturthur or @scott-es :)

File: excludarr/core/sonarr_actions.py [hunk 1]
```diff
@@ -140,24 +134,31 @@ def _find_serie(self, serie, jw_providers, tmdb_api_key, fast, exclude):
...
+                show = self._find_using_tvdb_id(
+                    title, sonarr_id, tvdb_id, fast
+                )
         elif tvdb_id and tmdb_api_key:
             # If the user has filled in an TMDB ID fall back to querying TMDB API using the TVDB ID
-            jw_id, jw_serie_data = self._find_using_tvdb_id(
+            show = self._find_using_tvdb_id(
                 title, sonarr_id, tvdb_id, fast, jw_query_payload
             )
+            print("kek")
```
Comment (TheArturthur) [inline excludarr/core/sonarr_actions.py:150]: This print statement should be removed, though it's not urgent.

**Review by TheArturthur — COMMENTED** [review]
All good, should also update Dockerfile for new python minimum version (3.12 instead of 3.11)

**Comment by TheArturthur** [comment]
@barsa2000 why did you close the PR so soon?
Sorry for the delay on my part. I, at least, work and have other responsibilities. Please, don't assume this repo is unmaintained because a PR wasn't reviewed in 3 days...
I will update my notifications configuration for the future, and probably set up alerts for admins to review. Any other config suggestions are welcome.

**Comment by barsa2000** [comment]
There are 2 maintainers and none of you were able to acknowledge the 2 PRs I made, I wanted to add enhancements to the tools but those PRs were blocking and I didn't really feel like waiting 2 weeks for a merge and having to change and rebase all the enhancements. So I just forked the original and made my own fork with other stuff. 

I work too and I work on this tool on my free time (you can see that pretty much all the commits were made at like 3am lol). If after one week there are no commits, no answers to issues and no pr reviews to me it looks like an abandoned project🤷‍♀️

**Comment by barsa2000** [comment]
You can see that before creating my own fork I waited about one week after the PRs

**Comment by barsa2000** [comment]
@TheArturthur if you want we can merge my fork with this and keep it under the excludarr org, it's not a big deal + I have added a lot of stuff and I'm working on making it a django web service

https://github.com/barsa2000/excludarr

**Comment by TheArturthur** [comment]
Fine by me. Let me check it out while you accept the org invite I sent you :)

**Comment by barsa2000** [comment]
Ok, then I'm gonna create a new PR to merge my fork so we can all review it before merging. otherwise I would have to recover branches and mirror the PRs I did on my fork and it would be a pain tbh.

**Comment by barsa2000** [comment]
moved to #17

**Comment by barsa2000** [comment]
moved to #17

**Pull request closed without merging**